prepare/consumer-data.sql

```diff
@@ -173,11 +173,11 @@ create or replace task populate_reading_every_minute
 as
     call populate_reading();
 
--- Get some initial data in the readings table
-call populate_reading();
+-- If you would like the data to be populated on a schedule, you can run:
```

Comment:
Going to get pedantic with my documentation experience thus far:
MIke H advises that we replace all "would like"s with "want".

Reply:
updated.